packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -87,6 +102,97 @@ export default class PciProjectNewPaymentCtrl {
       });
   }
 
+  displayCucCloudMessage(type, suffix) {
+    this.CucCloudMessage[type](
+      this.$translate.instant(
+        `pci_project_new_payment_check_anti_fraud_case_${suffix}`,
+      ),
+      'pci.projects.new.payment',
+    );
+  }
+
+  displayAntiFraudMessage(validatingStep, order) {
+    const {
+      FRAUD_REFUSED,
+      FRAUD_DOCS_REQUESTED,
+      FRAUD_MANUAL_REVIEW,
+    } = ORDER_FOLLOW_UP_STEP_VALIDATING_HISTORY.ENUM;
+
+    (validatingStep?.history || []).forEach(({ label }) => {
+      if (label === FRAUD_REFUSED) {
+        this.displayCucCloudMessage('error', label.toLowerCase());
+      }
+      if ([FRAUD_MANUAL_REVIEW, FRAUD_DOCS_REQUESTED].includes(label)) {
+        this.orderBillingUrl = this.buildOrderBillingUrl(order);
+        this.needToCheckCustomerInformartions = true;
```

Comment:
```suggestion
        this.needToCheckCustomerInformations = true;
```

And everywhere it's used.

Reply:
Done